traceevents/src/main/java/com/tomtom/kotlin/traceevents/Tracer.kt

```diff
@@ -522,20 +593,70 @@ class Tracer private constructor(
             }
         }
 
-        internal fun createLogMessage(traceEvent: TraceEvent): String {
-            return "[${traceEvent.dateTime.format(DateTimeFormatter.ISO_DATE_TIME)}] " +
+        internal fun registeredToString(item: Any?) =
```

Comment:
This method reads a bit odd. I'd suggest starting method names with a verb. E.g.,
```suggestion
        internal fun convertToString(item: Any?) =
```
or
```suggestion
        internal fun convertToStringUsingRegistry(item: Any?) =
```

Reply:
Renamed,